Classes/View/PreviewView.php

```diff
@@ -482,6 +482,7 @@ protected function getRecords(PageLayoutView $view, array $row, $area) {
 		// records based on them having remove placeholders.
 		$condition = "AND tx_flux_parent = '" . $row['uid'] . "' AND tx_flux_column = '" . $area . "' ";
 		$condition .= "AND colPos = '" . ContentService::COLPOS_FLUXCONTENT . "' ";
+		$condition .= $view->tt_contentConfig['showHidden'] ? '' : 'AND hidden = 0 ';
```

Comment:
Please write a compare like TRUE ===
Also you can use a Elvis operator ?: 'AND hidden = 0 '